### PR TITLE
Register the json_symtab languauge in symtab2gb

### PR DESCRIPTION
**Pull request by martin-cs**
To quote langapi/mode.cpp:

"Note: registering a language is required for using the functions
in language_util.h"

symtab2gb previously failed to do this and so fails in invariant
under some obscure and unclear circumstances.

- [ ] Each commit message has a non-empty body, explaining why the change was made.
- [ ] Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- [ ] The feature or user visible behaviour I have added or modified has been documented in the User Guide in doc/cprover-manual/
- [ ] Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- [ ] My commit message includes data points confirming performance improvements (if claimed).
- [ ] My PR is restricted to a single feature or bugfix.
- [ ] White-space or formatting changes outside the feature-related changed lines are in commits of their own.

### CODE REVIEW

**Comment by martin-cs** [comment]
Unfortunately I don't have access to the test case that triggers this but as it is a violation of our own documentation, I figure it is clearly enough a bug.

File: src/symtab2gb/symtab2gb_parse_options.cpp [hunk 1]
```diff
@@ -111,6 +111,7 @@ int symtab2gb_parse_optionst::doit()
   {
     gb_filename = cmdline.get_value(SYMTAB2GB_OUT_FILE_OPT);
   }
+  register_language(new_json_symtab_language);
```
Comment (tautschnig) [inline src/symtab2gb/symtab2gb_parse_options.cpp:115]: Failing build note: `symtab2gb_parse_options.cpp:114:3: error: ‘register_language’ was not declared in this scope`

Comment (martin-cs) [reply src/symtab2gb/symtab2gb_parse_options.cpp:115]: Ooops.  Should be fixed now.

**Comment by martin-cs** [comment]
I am not sure how the over-all coverage is decreasing if the patch has 100% coverage.

**Review by TGWDB — APPROVED** [review]

**Pull request merged**